examples/hybrid/src/web/mod.rs

```diff
@@ -0,0 +1,14 @@
+use axum::routing::get;
+
+// basic handler that responds with a static string
+async fn root() -> &'static str {
+    "Hello, World!"
+}
+
+pub struct ApiService;
+
+impl ApiService {
+    pub fn build() -> axum::Router {
+        axum::Router::new().route("/", get(root).post(root))
```

Comment:
I think we should just put this line in `main`. No reason to have `ApiService` since it doesn't have any state and only have inherent methods.